### DISCUSSION TITLE
Remove fleet role from Manager

### DISCUSSION THREAD

**Discussion by securityfai** [Security-Onion-Solutions/securityonion#13901] [2.4] [answered]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

125

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hi, 
I have a distributed Security Onion architecture with a dedicated Fleet node. I want to remove the Fleet role from the manager so that agents only attempt to enroll with this dedicated Fleet node, rather than trying to connect to multiple Fleet instances (in this case, the manager and the Fleet node). Could someone confirm if this is possible, and if so, how to make it happen?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment, marked as answer]
The docs cover there here: https://docs.securityonion.net/en/2.4/elastic-fleet.html#settings
